html-syntax/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+  <img src = "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTf1K5Qg5LP4ziymf965yCPgtxrwpCNc2q-Yg&usqp=CAU"
+  alt= " Bulbasaur" title = "Picture of Bulbasaur"
+  />
```

Comment:
When creating attributes on HTML elements, there should not be any spaces before or after the equal sign. A minor note, but an important one!

Reply:
![Screen Shot 2021-06-21 at 2 48 40 PM](https://user-images.githubusercontent.com/52221756/122832180-daf42b00-d29f-11eb-83f2-ed1d27e33e20.png)